build.sbt

```diff
@@ -1,15 +1,15 @@
-organization in Global := "io.backchat.inflector"
-homepage in Global := Some(url("https://github.com/backchatio/scala-inflector"))
+organization in Global := "com.qvantel"
+homepage in Global := Some(url("https://github.com/qvantel/scala-inflector"))
 startYear in Global := Some(2010)
-licenses in Global := Seq(("MIT", url("http://github.com/backchatio/scala-inflector/raw/HEAD/LICENSE")))
+licenses in Global := Seq(("MIT", url("https://github.com/qvantel/scala-inflector/raw/HEAD/LICENSE")))
 
-version in Global := "1.3.6-SNAPSHOT"
+version in Global := "1.3.6"
 
-scalaVersion in Global := "2.12.0"
-crossScalaVersions in Global := Seq(scalaVersion.value, "2.11.8")
+scalaVersion in Global := "2.12.3"
+crossScalaVersions in Global := Seq(scalaVersion.value, "2.11.11")
 
 scalacOptions in Global ++= Seq("-unchecked", "-deprecation", "-Xcheckinit", "-encoding", "utf8")
-scalacOptions in Global ++= (if (scalaVersion.value startsWith "2.12.") Seq("-opt:_") else Seq("-optimize"))
+scalacOptions in Global ++= (if (scalaVersion.value startsWith "2.12.") Seq("-opt:l:inline", "-opt-inline-from") else Seq("-optimize"))
```

Comment:
Not sure if `-opt-inline-from` by itself does anything? See the link in the [release announcement](http://scala-lang.org/news/2.12.3).

This also disables the other optimizations. You may want to use something like:

```scala
"-opt:-l:classpath,-l:project,l:inline,_",
"-opt-inline-from:<whatever>",
```